.github/workflows/python-build-workflow.yml

```diff
@@ -0,0 +1,30 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install poetry
+          make install
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: lint fix
+        run: | 
+         make lint-fix
```

Comment:
`lint-fix` is intended to be run by developers prior to check-in to fix any automatically fixable issues, not as a build step.